qemu_v8.mk

```diff
@@ -133,7 +133,7 @@ linux-cleaner: linux-cleaner-common
 # OP-TEE
 ################################################################################
 OPTEE_OS_COMMON_FLAGS += PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y \
-			 DEBUG=0 CFG_PM_DEBUG=0
+			 DEBUG=$(DEBUG) CFG_PM_DEBUG=0
```

Comment:
by the way, `CFG_PM_DEBUG` is a deprecated thing, should be removed.

Reply:
Yes I was not sure about that that's why I didn't change it.
I will also remove it then.